.github/labeler.yml

```diff
@@ -1,104 +1,69 @@
+"area: crates":
+  - "crates/**/*"
```

Comment:
We want to rename `crates` to `libs` in line with our agreed [monorepo structuring principles](https://www.notion.so/hashintel/Monorepo-Structure-216b3150cfdf48bb9f1bc149dd25861b), and additionally move the contents of `packages` in here (except where things may in fact be `apps`).

Reply:
The goal of this PR is to sync existing labels with existing repo structure. I suggest syncing labels again ones (if?) `crates` and `packages` are put under `lib`. Might be worth checking this idea again with @CiaranMn and @Alfred-Mountfield in early Oct.

Reply:
There's no 'if', as our monorepo structure/principles were agreed. Unless there's new information we didn't have then, let's not re-open that finished discussion. Happy for this change to be made separately, but it does need making.